src/case-ui-toolkit.module.ts

```diff
@@ -108,7 +108,6 @@ import { CaseHistoryModule } from './shared/components/case-history';
       WriteEmailFieldComponent,
       WriteDateFieldComponent,
 
-      ConditionalShowDirective,
```

Comment:
I may be wrong, but I think removing this would constitute a breaking change. MC is not the only consumer of the CCD Case UI Toolkit and if anything else is leveraging that, it'll break. It's also not needed for the change you're making.